croutonjs/src/js/crouton-core.js

```diff
@@ -1059,6 +1060,39 @@ crouton_Handlebars.registerHelper('formatDataPointer', function(str) {
 	return convertToPunyCode(str)
 });
 
+crouton_Handlebars.registerHelper('isVirtual', function(data, options) {
+	var fnTrue = options.fn;
+	var fnFalse = options.inverse;
+
+	return inArray('VM', data['formats'].split(",")) && (data['virtual_meeting_link'] || data['phone_meeting_number']) ? fnTrue(this): fnFalse(this);
+});
+
+crouton_Handlebars.registerHelper('isTemporarilyClosed', function(data, options) {
+	var fnTrue = options.fn;
+	var fnFalse = options.inverse;
+
+	return inArray('TC', data['formats'].split(",")) ? fnTrue(this): fnFalse(this)
+});
+
+crouton_Handlebars.registerHelper('isNotTemporarilyClosed', function(data, options) {
+	var fnTrue = options.fn;
+	var fnFalse = options.inverse;
+
+	return !inArray('TC', data['formats'].split(",")) ? fnTrue(this): fnFalse(this)
+});
+
+crouton_Handlebars.registerHelper('temporarilyClosed', function(data, options) {
+	if (data['formats_expanded'].getArrayItemByObjectKeyValue('key', 'TC') !== undefined) {
+		return data['formats_expanded'].getArrayItemByObjectKeyValue('key', 'TC')['description'];
+	} else {
+		return "TEMPORARILY CLOSED";
```

Comment:
Under what conditions will this else block be hit?

Reply:
The condition occurs when there is a TC code, but no translations filled in.  It falls back to something.